<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_artikel</class>
 <widget class="QDialog" name="Dialog_artikel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Artikel</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Artikelnummer:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Lieferant</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Bezeichnung</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_artielnummer"/>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBox_lieferant"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_bezeichnung"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_filter_lieferant"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_filter_lieferant">
      <property name="text">
       <string>Filter Lieferanten</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_lagerort">
      <property name="text">
       <string>Lagerort</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_ve">
      <property name="text">
       <string>VE</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_komment">
      <property name="text">
       <string>Kommentar</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_lagerort"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_komment"/>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="spinBox_ve">
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>381</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_artielnummer</tabstop>
  <tabstop>lineEdit_bezeichnung</tabstop>
  <tabstop>lineEdit_filter_lieferant</tabstop>
  <tabstop>comboBox_lieferant</tabstop>
  <tabstop>lineEdit_lagerort</tabstop>
  <tabstop>spinBox_ve</tabstop>
  <tabstop>lineEdit_komment</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
